### DISCUSSION TITLE
Enabling the TimelineTab on 4.1.x

### DISCUSSION THREAD

**Discussion by reacan** [sepinf-inc/IPED#1720] [Q&A] [answered]
Hello!

I am running IPED 4.1.2 (on Windows and Linux) and I do not see a "Timeline" tab in UI. As far as I know it should be visible between "Map" and "Links" tabs. 

I did not see any errors when compiling. There are no errors in processing logs and analysis app logs as well. I double-checked all config files to see if there are any options to enable the "Timeline" tab but could not find anything.

How do I enable the "Timeline" tab?

**Comment by lfcnassif** [comment, marked as answer]
There was a fix on https://github.com/sepinf-inc/IPED/issues/1565 where user custom layouts were preventing the new tab to appear, but it should have been fixed by 4.1.1.

Please check your `conf/ResultSetViewersConf.xml` file if it has all entries below. Maybe you are using a custom profile with an old version of that file.
```
<resultSetViewers>
	<resultSetViewer>
		<class>iped.geo.impl.MapViewer</class>
	</resultSetViewer>
	<resultSetViewer>
		<class>iped.app.timelinegraph.IpedChartsPanel</class>
	</resultSetViewer>
</resultSetViewers>
```

**Reply by reacan** [reply]
I updated the config file and added the fix #1707 and everything is working now. Thank you very much!
